mindsdb/interfaces/model/model_controller.py

```diff
@@ -213,3 +248,118 @@ def import_predictor(self, name: str, payload: json, company_id: int) -> None:
             fp.write(predictor_binary)
 
         self.fs_store.push()
+
+    def process_create_statement(self, statement, database_controller, handler_controller):
+        # TODO use database_controller handler_controller internally
```

Comment:
Please:
1. provide docstrings in google styleguide format https://google.github.io/styleguide/pyguide.html#s3.8-comments-and-docstrings.
2. provide type of input parameters. you may do in direct in function signature or in the doc strings.